### PR TITLE
camunda-modeler: init at 5.13.0

### PR DESCRIPTION
**Pull request by n0emis**
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by hexchen — APPROVED** [review]
Looks good to me, derivation builds and the application starts on my machine.

**Comment by johannwagner** [comment]
We are using this for some weeks now on an amd64 Linux and aarch64 Darwin machine and it works flawlessly. 🔥

File: pkgs/applications/misc/camunda-modeler/default.nix [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  dontBuild = true;
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin $out/share/${pname} $out/share/applications $
+    cp -a {locales,resources} $out/share/${pname}
+    ln -s "${desktopItem}/share/applications/${pname}.desktop" "$out/share/applications/"
```
Comment (yu-re-ka) [inline pkgs/applications/misc/camunda-modeler/default.nix:29]: Please use `copyDesktopItems` hook by adding it to nativeBuildInputs and doing

```
  desktopItems = [
    (makeDesktopItem {
       [...]```

**Comment by yorickvP** [comment]
@ofborg camunda-modeler

**Comment by yorickvP** [comment]
@ofborg build camunda-modeler

**Review by kip93 — APPROVED** [review]
LGTM

File: pkgs/applications/misc/camunda-modeler/default.nix [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+, copyDesktopItems
+}:
+
+stdenv.mkDerivation rec {
+  pname = "camunda-modeler";
+  version = "5.13.0";
+
+  src = fetchurl {
+    url = "https://github.com/camunda/camunda-modeler/releases/download/v${version}/camunda-modeler-${version}-linux-x64.tar.gz";
+    sha256 = "sha256-/9Af/1ZP2Hkc0PP9yXObNDNmxe6riBNWSv+JaM7O5Vs=";
```
Comment (wegank) [inline pkgs/applications/misc/camunda-modeler/default.nix:16]:
```suggestion
    hash = "sha256-/9Af/1ZP2Hkc0PP9yXObNDNmxe6riBNWSv+JaM7O5Vs=";
```

File: pkgs/applications/misc/camunda-modeler/default.nix [hunk 3]
```diff
@@ -0,0 +1,71 @@
+{ stdenv
+, lib
+, fetchurl
+, electron
+, makeWrapper
+, makeDesktopItem
+, copyDesktopItems
+}:
+
+stdenv.mkDerivation rec {
```
Comment (wegank) [inline pkgs/applications/misc/camunda-modeler/default.nix:10]:
```suggestion
stdenvNoCC.mkDerivation rec {
```

**Pull request merged**